src/server/metrics.rs

```diff
@@ -49,7 +49,7 @@ make_auto_flush_static_metric! {
         remove_lock_observer,
         coprocessor,
         coprocessor_stream,
-        coprocessor_v2,
+        raw_coprocessor,
```

Comment:
In some lines, we now have `raw_coprocessor`, in others `copr_v2`. Is this confusing?

Reply:
`copr_v2` plugin is able to handle both `raw_coprocessor` and `txn_coprocessor`. So at the request level, I named it `RawCoprocessor`, and at the plugin level, it's called `copr_v2`.

Reply:
makes sense! thanks!